components/Settings/ClubTab.js

```diff
@@ -0,0 +1,87 @@
+import { DARK_GRAY } from '../../constants/colors'
+import { CLUB_ROUTE } from '../../constants/routes'
+import s from 'styled-components'
+import Toggle from './Toggle'
+import { Icon } from '../common'
+import ReactTooltip from 'react-tooltip'
+
+const Table = s.table`
+  font-family: HelveticaNeue;
```

Comment:
Also would be good if fonts are a constant in another file (just like how colors are); then we could say:
```
font-family: ${BODY_FONT};
```
And can easily change fonts / not misspell them